src/container/container.ts

```diff
@@ -108,11 +108,29 @@ class Container implements interfaces.Container {
             };
         };
 
+        let getRebindFunction = (moduleId: string) => {
+            return (serviceIdentifier: interfaces.ServiceIdentifier<any>) => {
+                let _rebind = this.rebind.bind(this);
```

Comment:
:+1: thanks a lot for the review :100: I have done those changes at https://github.com/inversify/InversifyJS/pull/491